index.bs

```diff
@@ -4863,9 +4862,9 @@ partial interface MLGraphBuilder {
   <div class=algorithm-steps>
     1. [=Assert=]: the type of |input| is {{MLOperand}}.
     1. Let |outputShape| be an empty array of {{unsigned long}}.
-    1. If |newShape| is a scalar [=number=], set |outputShape| to `«  1  »`.
+    1. If |newShape| is a scalar [=number=], set |outputShape| to `«»`.
```

Comment:
|newShape| should not be a number. We should assert that |newShape| is an array of unsigned long.

Reply:
Does `[=number=]` in this weird syntax refer to a JS `Number`? I presume that be changed to `{{unsigned long long}}` then (I don't get the significance of of `[[` vs `[=`, but `{{unsigned long long}}` is what I see being used elsewhere).

Reply:
Yes, `[=number=]` links to JS `Number`.  Looks like we need to formalize using `{{unsigned long long}}`, eventually with a typedef, and consider including it in normative steps as well, not only asserts (test anchors).

Reply:
> Yes, [=number=] links to JS Number

👍

> eventually with a typedef

I'll defer that to someone else's later change, as I want this change to be surgical.

Reply:
Ningxin: Doh, merge conflict. Working on fixing this line. ⏳